docker/plugin/digitile-monstache-plugin.go

```diff
@@ -79,7 +80,7 @@ func ProcessDocument(input *monstachemap.MapperPluginInput) (output *monstachema
 
 	defer func() {
 		if recoveredError := recover(); recoveredError != nil {
-			err = fmt.Errorf("Syncing asset with ID %s failed with an unknow error: %s",
+			err = fmt.Errorf("Syncing asset with ID %s failed with an unknown error: %s",
 				assetId, recoveredError)
 		}
 	}()
```

Comment:
This defer function only handle panics within the function that contains it and not in goroutines called from this function.